### ISSUE TITLE
LUA errors and cannot set item value

### ISSUE THREAD

**Issue by Armonth** [E1ila/FarmLog#1]
Everytime I click on the button I get this message:

```
Message: Interface\AddOns\FarmLog\FarmLog.lua:106: Usage: GetCoinText(amount, fontHeight)
Time: Sat Oct 19 16:46:49 2019
Count: 1
Stack: Interface\AddOns\FarmLog\FarmLog.lua:106: Usage: GetCoinText(amount, fontHeight)
[C]: in function `GetCoinTextureString'
Interface\AddOns\FarmLog\FarmLog.lua:106: in function <Interface\AddOns\FarmLog\FarmLog.lua:98>
Interface\AddOns\FarmLog\FarmLog.lua:482: in function `AddSessionYieldItems'
Interface\AddOns\FarmLog\FarmLog.lua:541: in function `FLogRefreshSChildFrame'
Interface\AddOns\FarmLog\FarmLog.lua:1014: in function <Interface\AddOns\FarmLog\FarmLog.lua:1002>
Interface\AddOns\FarmLog\FarmLog.lua:1164: in function <Interface\AddOns\FarmLog\FarmLog.lua:1164>

Locals: (*temporary) = Infinite
(*temporary) = 12

```
In anycase, the addon seems to work well except this error every time I press the minimap button.

That and the /fl set i'm not sure if is my fault (making something wrong) but cannot set a item value to X value, what i am doing is target with the cursor a item and typing /fl set 15. The error I get is:

```
Message: Interface\AddOns\FarmLog\FarmLog.lua:1848: bad argument #2 to 'sub' (number expected, got nil)
Time: Sat Oct 19 16:54:57 2019
Count: 2
Stack: Interface\AddOns\FarmLog\FarmLog.lua:1848: bad argument #2 to 'sub' (number expected, got nil)
[C]: in function `sub'
Interface\AddOns\FarmLog\FarmLog.lua:1848: in function `?'
Interface\FrameXML\ChatFrame.lua:4609: in function `ChatEdit_ParseText'
Interface\FrameXML\ChatFrame.lua:4270: in function `ChatEdit_SendText'
Interface\FrameXML\ChatFrame.lua:4306: in function `ChatEdit_OnEnterPressed'
[string "*:OnEnterPressed"]:1: in function <[string "*:OnEnterPressed"]:1>

Locals: (*temporary) = "15"
(*temporary) = nil
(*temporary) = nil
(*temporary) = "number expected, got nil"

```

**Comment by E1ila** [comment]
to set AH value for an item, you should write `/fl set [Item Link] 5.5`, then [Item Link] price will be set to 5.5 gold. For [Item Link] use shift and click the item from your bag to put the item link in the chat box.

**Comment by E1ila** [comment]
`GetCoinText` error fixed in commit a92618d
